docker/build.sh

```diff
@@ -140,7 +142,7 @@ source container-test.sh
 for arch in "${ARCHITECTURES[@]}"; do
     # TODO: remove when https://github.com/multiarch/qemu-user-static/issues/128 is fixed
     if [[ "$arch" != "ppc64le" ]]; then
-        test_container "${TAG_PREFIX}-$arch" "linux/$arch" "${SCRIPT_DIR}/minimal-compose.yml" "gateway" "postgres"
+        test_container "${TAG_PREFIX}-$arch" "linux/$arch" "${SCRIPT_DIR}/docker-compose.yml" "gateway" "postgres"
```

Comment:
This script is broken. Rename gateway to trino-gateway.

Reply:
Fixed

Reply:
Renamed passed parameter and section in chart. Testing again.